ci/__main__.py

```diff
@@ -1,17 +1,16 @@
 import click
 from subprocess import check_call
 
-DEFAULT_PYTHON_VERSION = "3.5"
-PYTHON_VERSIONS = ["3.5"]
+DEFAULT_PYTHON_VERSION = "3.6"
+PYTHON_VERSIONS = ["3.6"]
 
 ADDITIONAL_CORE_DEPS = [
-    "pyface==6.0.0-1",
+    "pyface==6.0.0-2",
     "pygments==2.2.0-1",
     "pyqt==4.11.4-7",
     "qt==4.8.7-7",
     "sip==4.17-4",
-    "traits==4.6.0-1",
```

Comment:
I'm not actually sure what the standard way to do this is, but could we ask for traits 5.0.0-1 in this list? It gets pulled in as a dependency right now, but I think that may mean it gets the latest one by default. Which I'd imagine is fine almost all the time, but could jump in as a 'silent' error if a newer version of traits breaks something in future

Reply:
I guess my thinking was that force_bdss is an unlisted dependency for this, which will have bundled the version of Traits that it needs. Perhaps we should include force_bdss here instead?

Reply:
That would be nice! But I think 'cross-linking' the build scripts might be harder to maintain at this stage since they don't have proper edm/pip egg distributions. I really don't know much about this stuff though.
I think traits 5 actually gets pulled in as a dependency of envisage in force_bdss in any case!
So I'd be happy to keep the deletion, I guess also the less requirements you put on edm the easier it is for it to resolve the dependencies!